packages/block-library/src/navigation/index.php

```diff
@@ -5,6 +5,120 @@
  * @package WordPress
  */
 
+// These functions are used for the __unstableLocation feature and only active
+// when the gutenberg plugin is active.
+if ( defined( 'IS_GUTENBERG_PLUGIN' ) ) {
```

Comment:
```suggestion
if ( defined( 'IS_GUTENBERG_PLUGIN' )  &&  IS_GUTENBERG_PLUGIN ) {
```

See my rational for this change in this comment https://github.com/WordPress/gutenberg/pull/37021/files#r761303666

Reply:
I don't see code from this file in Core. Is this code intended to be backported to WordPress Core? Or is it Gutenberg plugin only code? If only for the plugin, then the guard check is not needed.

cc @adamziel @talldan @noisysocks